### ISSUE TITLE
urllib3 is missing for requirements

### ISSUE THREAD

**Issue by jimoleary** [victorhooi/mongo-insight#11]
title says it all

**Comment by victorhooi** [comment]
Yeah, I think I added in an explicit import for that, to try and disable the self-signed SSL certificates warnings.

It didn't work with the present code (https://github.com/influxdb/influxdb-python/issues/240), so I'm going to try the suggested alternative there - which means we don't need to import urllib3 explicitly. Will update.
